cis/user.py

```diff
@@ -56,8 +56,26 @@ def store_in_vault(self):
         if not self.dynamodb_table:
             self._connect_dynamo_db()
 
+        # Workaround DynamoDB limitation
+        # Replace `None` values with empty string
+
+        def update_values(d):
+            """Recursively update None values with empty string."""
+            new = {}
+            for k in d.keys():
+                v = d[k]
+                if isinstance(v, dict):
+                    v = update_values(v)
+                if v is None:
+                    new[k] = ''
```

Comment:
Please change to `Null` we can revisit this choice late 2018 when DynamoDb supports empty string type.